tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java

```diff
@@ -202,6 +202,16 @@ protected ModuleContainer newState(DependencyArtifacts artifacts, Properties pro
             throws BundleException {
         ModuleContainer[] moduleContainerAccessor = new ModuleContainer[1];
         ModuleContainerAdaptor moduleContainerAdaptor = new ModuleContainerAdaptor() {
+
+            @Override
+            public String getProperty(String key) {
+                switch (key) {
+                case "equinox.resolver.revision.batch.size":
+                    return Integer.toString(1);
```

Comment:
Maybe we could replace this by System.getProperty("tycho.resolver.batch.size", "1") to allow configuration through system properties.

Reply:
I didn't add it. More options is more complexity and are only worth the effort when we know we need them.